### PR TITLE
PR-621 => develop

### PR DESCRIPTION
**Pull request by nmsger-isento**
# Jira Ticket:

Link to corresponding jira ticket:
https://pib-rocks.atlassian.net/jira/software/c/projects/PR/boards/3?selectedIssue=PR-621

# Checklist:

Necessary tasks before pull request can be merged:

-   [x] The code has been reviewed and approved at least once
-   [x] All necessary tests have been successful
-   [x] The product owner reviewed and accepted the story

# Testing:

### Test acceptance criteria:

-   [x] Cerebra works as usual

### Setup instructions:

Run setup script with `-f=PR-621`

-   [x] Setup on a physical pib was successful

### CODE REVIEW

File: src/app/voice-assistant/voice-assistant-chat/chat-window/chat-window.component.scss [hunk 1]
```diff
@@ -116,3 +116,9 @@
     outline: none;
     opacity: 0.3;
 }
+
```
Comment (nisento) [inline src/app/voice-assistant/voice-assistant-chat/chat-window/chat-window.component.scss:119]: why did you add this css?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -100,5 +100,6 @@ import {PoseComponent} from "./joint-control/pose/pose.component";
         },
     ],
     bootstrap: [AppComponent],
+    // entryComponents: [MatDialogModule],
```
Comment (nisento) [inline src/app/app.module.ts:103]: do we need this?

**Review by nisento — CHANGES_REQUESTED** [review]

File: src/app/voice-assistant/voice-assistant-chat/chat-window/chat-window.component.scss (hunk 1 above)
Comment (nmsger-isento) [reply src/app/voice-assistant/voice-assistant-chat/chat-window/chat-window.component.scss]: That was not supposed to be part of this PR, so I removed it again.

File: src/app/app.module.ts (hunk 2 above)
Comment (nmsger-isento) [reply src/app/app.module.ts]: Due to the migration to Angular 18, not anymore. I removed the comment.

**Review by nisento — APPROVED** [review]

**Review by niklasIsento — APPROVED** [review]
